### ISSUE TITLE
The color of the description foreground in content assistant popups can be improved for accessibility concern

### ISSUE THREAD

**Issue by jdneo** [eclipse-platform/eclipse.platform.ui#3468]
According to the app 'Accessibility Insight for Windows', it shows that the color ratio between the description foreground and background is less than 4.5:1. See:

<img width="1420" height="834" alt="Image" src="https://github.com/user-attachments/assets/29ab4658-69a2-4aa6-b965-4603446ae491" />

We can make it slightly darker, for example. change it from `#808080` to `#767676`, to fulfill the requirement:

<img width="578" height="537" alt="Image" src="https://github.com/user-attachments/assets/d110f8e8-c8b3-4ab4-b7c7-49d2d8a1c5d3" />

Reference: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

**Comment by BeckerWdf** [comment]
I am not sure if I get what you say. Do you refer to the contrast ration between the background color of the selected item in the content assist (grey) and the text of it (black)?
But you measured the contrast between the grey and white.
Can you explain in more detail?

**Comment by jdneo** [comment]
I'm measuring this part

<img width="276" height="96" alt="Image" src="https://github.com/user-attachments/assets/8c25d982-db63-40f3-9903-6c54b9a7b013" />

**Comment by BeckerWdf** [comment]
Can you also check in the dark theme? 
Is the same color used here or does the Description use a specific color for the dark theme?
How is the contrast there?

**Comment by jdneo** [comment]
Yes same color used in dark mode, and it's more worse

<img width="1469" height="1025" alt="Image" src="https://github.com/user-attachments/assets/71f07ef9-7b65-44a8-aec1-5864684d714f" />

If set it to #A1A1A1 looks good

<img width="729" height="876" alt="Image" src="https://github.com/user-attachments/assets/83d84863-5733-4960-8b4b-78a422b2a553" />

**Comment by BeckerWdf** [comment]
this color is coming from `JFacePreferences#QUALIFIER_COLOR`
Which can be configures here:

<img width="674" height="690" alt="Image" src="https://github.com/user-attachments/assets/947ae2cd-0b6d-4dfd-b324-b289d363f1bb" />

So to fix this issue we can define a better default and also provide a separate color for the dark theme in the dark theme's CSS file.

This color is also used on multiple other places in the IDE. But the good thing is it's always on a light background (in the light theme)

I set this to magenta in my IDE so it can easily be seen:

It's used e.g. in the Git Repo View:
<img width="547" height="89" alt="Image" src="https://github.com/user-attachments/assets/ecc1f6da-93da-4e5e-8118-3627edcfc2c1" />

Used in "Find actions" (Command-3) so render the key bindings for commands:

<img width="663" height="127" alt="Image" src="https://github.com/user-attachments/assets/51220ee9-e0c5-44ae-bfa8-0432a7b3d889" />

In the open Resource dialog:

<img width="874" height="545" alt="Image" src="https://github.com/user-attachments/assets/33a2968d-c035-4afe-8177-b9397f8a5585" />

Do we have white for the editor or dialog background and a very light grey as background  in views. So the contrast has to be good in both these cases.

Similarly in the dark theme where we have a darker background in the editor and and slightly lighter background in the views.
